waspc/data/Generator/templates/server/src/routes/auth/signup.js

```diff
@@ -1,11 +1,23 @@
 {{={= =}=}}
+import Prisma from '@prisma/client'
 import prisma from '../../dbClient.js'
 import { handleRejection } from '../../utils.js'
+import EntityValidationError from '../../core/EntityValidationError.js'
+import HttpError from '../../core/HttpError.js'
 
 export default handleRejection(async (req, res) => {
   const userFields = req.body || {}
 
-  await prisma.{= userEntityLower =}.create({ data: userFields })
+  try {
+    await prisma.{= userEntityLower =}.create({ data: userFields })
+    res.send()
```

Comment:
Would it make sense to pull this line out of try/catch? I know this will not affect the execution since you are forwarding errors anyway, but it will make it clearer that we are handling errors from the line 12 and are not interested in line 13 here.

Reply:
Good point 👍🏻 Thanks!

Reply:
This has been fixed here: https://github.com/wasp-lang/wasp/pull/376/commits/b2b086cdee30c3c8ea75786698cc0aff40b8f1f1 thanks!